### PR TITLE
docs(daemon): refactor remote executors

### PR DESCRIPTION
**Pull request by deepankarm**
Fixes https://github.com/jina-ai/internal-tasks/issues/174

### CODE REVIEW

File: docs/advanced/daemon/remote-executors.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Remote Executor
+
+JinaD enables deploying Executors on remote machines. We can pass the remote info using `host`and `port_jinad` arguments in the Flow syntax. Here are a few examples of using remote Executors.
+
+
+### Executor defined in yaml
```
Comment (alexcg1) [inline docs/advanced/daemon/remote-executors.md:6]: - YAML

File: docs/fundamentals/flow/add-exec-to-flow.md [hunk 2]
```diff
@@ -137,40 +137,7 @@ f.add(host='localhost', port_in=12345, external=True)
 f.add(host='123.45.67.89', port_in=12345, external=True)
 ```
 
-### Add & spawn a remote Executor via `jinad`
```
Comment (alexcg1) [inline docs/fundamentals/flow/add-exec-to-flow.md:140]: - & -> and

File: docs/fundamentals/flow/add-exec-to-flow.md [hunk 3]
```diff
@@ -137,40 +137,7 @@ f.add(host='localhost', port_in=12345, external=True)
...
-
-In the example below, the Executor with
-the `host`
-keyword `gpu-exec`, is put to a remote machine for parallelization, whereas other Executors stay local. Extra file
```
Comment (alexcg1) [inline docs/fundamentals/flow/add-exec-to-flow.md:144]: - stay**s** local
- put ~to~ **on**

**Review by alexcg1 — CHANGES_REQUESTED** [review]
Looking good! Just a few tiny changes

File: docs/fundamentals/flow/add-exec-to-flow.md (hunk 2 above)
Comment (deepankarm) [reply docs/fundamentals/flow/add-exec-to-flow.md:140]: This section is removed now.

File: docs/fundamentals/flow/add-exec-to-flow.md (hunk 3 above)
Comment (deepankarm) [reply docs/fundamentals/flow/add-exec-to-flow.md:144]: same

**Review by hanxiao — APPROVED** [review]

**Pull request merged**